docs/plugin-proposal-pipeline-operator.md

```diff
@@ -18,10 +18,18 @@ $ npm install @babel/plugin-proposal-pipeline-operator
 
 ```json
 {
-  "plugins": ["@babel/plugin-proposal-pipeline-operator"]
+  "plugins": [["@babel/plugin-proposal-pipeline-operator", { "proposal": "minimal" }]]
 }
 ```
 
+The Pipeline Proposal is one of three competing implementations. Which implementation the plugin should use is configured with the `"proposal"` option. This option is required and should be one of:
```

Comment:
general question: should we start adding versions in docs? Like mention this is only the case for >= beta.51, and only merge docs PRs after the release?

Reply:
Do we need to specify by beta version for something like this?